Green command line: add tags (#525)
NabuCasa/support: src/green/troubleshooting/using-the-console.md

## Prerequisites

{% steps %}
{% prereq %}
{% image "/static/img/green/green_yellow_heartbeat.webp" "Image showing the Home Assistant Green with a yellow heartbeat" %}
{% stepContent %}

- Connected Home Assistant Green
- External USB keyboard
- HDMI cable and external monitor

{% endstepContent %}
{% endprereq %}
{% endsteps %}

## To use the console on the HDMI display

{% steps %}

> {% step "Connecting an HDMI monitor" %}

Author: ```suggestion
{% step "Connecting a monitor via HDMI" %}
```

{% image "/static/img/green/icons/hdmi-monitor.png" "HDMI monitor connected to Home Assistant Green" %}
{% stepContent %}

> - Connect an HDMI monitor to Home Assistant Green.

Reviewer: HDMI is the interface for the cable, not the monitor?

> - Connect an HDMI monitor to Home Assistant Green.

Author: ```suggestion
- Connect a monitor via HDMI to Home Assistant Green.
```

{% endstepContent %}
{% endstep %}

> {% step " Connecting a USB keyboard" %}

Reviewer: ```suggestion
{% step "Connecting a USB keyboard" %}
```
